packages/cli/src/handlers/download.ts

```diff
@@ -248,21 +316,19 @@ export const downloadContent = async (
                 [],
             );
         } else {
-            const outputDir = await createDirForContent(
-                contentAsCode.charts,
-                'charts',
-                customPath,
-            );
-            for (const [index, chart] of contentAsCode.charts.entries()) {
-                await writeContent(
-                    {
-                        type: 'chart',
-                        content: chart,
-                        translationMap: contentAsCode.languageMap?.[index],
-                    },
-                    outputDir,
+            const chartsBySpace = groupBySpace(contentAsCode.charts);
+            for (const [spaceSlug, chartsInSpace] of Object.entries(
+                chartsBySpace,
+            )) {
+                await writeSpaceContent({
```

Comment:
is the folder structure also creating spaces (with whate permissions?). How is this related to the `spaceSlug` field inside chart/dashboard. What happens if they are different ?